code/computational_geometry/src/bresenham_line/Bresenham_Line_Algorithm.java

```diff
@@ -0,0 +1,139 @@
+package applet;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.MouseMotionListener;
+
+public class bresenhamLine extends java.applet.Applet implements MouseListener, MouseMotionListener {
+    private static final long serialVersionUID = 1L;
+    int width;
+    int height;
+    int xa = 0,
+    ya = 0,
+    xb = 0,
+    yb = 0,
+    pixelsize = 2;
+
+    public void init() {
+        width = getSize().width;
+        height = getSize().height;
+        this.addMouseListener(this);
+        this.addMouseMotionListener(this);
+        }
+
+    // draw a line from point xa,ya into xb,yb
+    public void bresenham_line(int xa, int ya, int xb, int yb) 
```

Comment:
Follow a consistent naming convention for the member functions - camelcase, pascalcase etc. You could refer our Java style guide.